tests/input/fail_duplicate_definitions.fathom

```diff
@@ -1,3 +1,4 @@
-struct Empty {}
-struct Empty {} //~ error: the name `Empty` is defined multiple times
-struct Empty {} //~ error: the name `Empty` is defined multiple times
+struct Empty : Format {}
+struct Empty : Format {} //~ error: the name `Empty` is defined multiple times
+struct Empty : Format {} //~ error: the name `Empty` is defined multiple times
+Empty = 1 : Int; //~ error: the name `Empty` is defined multiple times
```

Comment:
Maybe it'd be helpful to have the user write `const Empty = 1 : Int;` when introducing this binding, otherwise it reads a bit like defining the type of `Empty` as `Format` and subsequently assigning a value to it.

Reply:
Made an issue for this here: #250

Reply:
Cool, thanks – i like being organised with cards we can work on :nerd_face: